plugins/ci/pkg/ci/ci.go

```diff
@@ -342,22 +339,27 @@ func SendResults(reports []models.ReportInfo, resources []models.Resource, confi
 func getGitInfo(repoName, baseBranch string) (gitInfo, error) {
 	info := gitInfo{}
 
+	_, err := GetResultsFromCommand("git", "rev-parse", "--is-inside-work-tree")
+	if err != nil {
+		logrus.Error("Unable to execute git command. Are you in a git repository?")
```

Comment:
@rbren I would rather do:

`return nil, fmt.Errorf("%v: %v", "Please be sure to run the insights-ci script inside of a valid git repository, with the branch you are scanning checked out.", err)`

here, what do you think? `getGitInfo` may fail for different reasons...

Reply:
Hmm yeah agreed!

Reply:
```INFO[0000] Updating dependencies for staging            
INFO[0000] Templating: staging                          
INFO[0000] Copying yaml file                            
INFO[0000] Copying yaml file                            
INFO[0000] Audit with Polaris                           
ERRO[0002] Unable to execute command: git rev-parse --is-inside-work-tree 
FATA[0002] Unable to get git details: Please be sure to run the insights-ci script inside of a valid git repository, with the branch you are scanning checked out: exit status 128 ```